docs/output.md

```diff
@@ -16,6 +16,7 @@ The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes d
     - [Trim adapters](#trim-adapters)
     - [Split FastQ files](#split-fastq-files)
     - [UMI consensus](#umi-consensus)
+    - [Bedtools](#bedtools)
```

Comment:
the bedtools output is changing the intervals file + there is only one per run/it's the same for all samples. IMO this should go to the reference directory, not the sample directories.

Reply:
Then, would the best option be Map to Reference?